static/app/views/performance/data.tsx

```diff
@@ -741,30 +741,31 @@ function generateFrontendOtherPerformanceEventView(
 }
 
 export function generatePerformanceEventView(
-  organization,
-  location,
-  projects,
-  isTrends = false
+  location: Location,
+  projects: Project[],
+  isTrends = false,
+  isMetricsData = false
```

Comment:
This was okay previously with just `isTrends` but with `isMetricsData`, this is getting confusing from the caller. Can we update the function signature to something like

```typescript
function generatePerformanceEventView(
  location: Location,
  projects: Project[],
  {isTrend, isMetricsData}: {isTrend: bool; isMetricsData: bool}
)
```

to make this clearer?